tests_e2e/scenarios/tests/error_test.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
```

Comment:
We have similar tests in DCR. They are very useful when debugging issues in the test pipeline.